scripts/release.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+ember build --production && npm publish
```

Comment:
Turn this into a JavaScript file and `execSync` and this will be portable. I'd prefer not to make the release process non-portable.